server/audit_math/macro.py

```diff
@@ -254,7 +259,7 @@ def compute_risk(
 
         p *= (1 - 1 / U) / (1 - taint)
 
-        if p < risk_limit:
-            return p, True
+        if p <= alpha:
+            return float(p), True
 
-    return p, p < risk_limit
+    return float(p), p <= alpha
```

Comment:
does `float(p)` pretty much round the way we want?

Reply:
yeah, since the issues with floats are way out at the end of the decimal places this should be fine. And really, those errors don't usually matter, except when they stack up when doing multiple multiplications.

Reply:
great!